openstack/dms/v2/products/requests.go

```diff
@@ -0,0 +1,20 @@
+package products
+
+import (
+	"github.com/chnsz/golangsdk"
+)
+
+// Get products
+func Get(client *golangsdk.ServiceClient, engine string) (*GetResponse, error) {
+	url := getURL(client)
+	url = url + "?engine=" + engine
```

Comment:
the URL is invalid if engine is empty.

Reply:
the URL is valid after verification, but I added the valid check